modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/selectionFilters.spec.ts

```diff
@@ -174,12 +262,17 @@ test.describe('Filters in Data Set Manager', () => {
 		page,
 		picklistApiHelpers,
 	}) => {
+		const picklist = await picklistApiHelpers.getPicklist(picklistName);
+		const preselectedValuesMultiSelect = page.locator(
+			'.form-control.form-control-tag-group.input-group'
```

Comment:
This is not a great locator, it does not include anything specific for preselected values select. It has chance of failing randomly later on, when we make an unrelated change on the page.

Reply:
I've changed the locator for a more accurate one, targeting the elements themselves inside the MultiSelect by label